libs/statistics/PALMastersMonteCarloValidation.h

```diff
@@ -102,14 +103,15 @@ template <class Decimal, class BaselineStatPolicy>
        * @throw PALMastersMonteCarloValidationException if numPermutations is zero.
        */
       PALMastersMonteCarloValidation(unsigned long numPermutations,
-				    std::unique_ptr<AlgoType> algo =
-				    std::make_unique<MastersRomanoWolfImproved<Decimal,BaselineStatPolicy>>())
-	: mNumPermutations(numPermutations),
-	  mStrategySelectionPolicy(),
-	  mAlgorithm(std::move(algo))
+        std::unique_ptr<AlgoType> algo =
+        std::make_unique<MastersRomanoWolfImproved<Decimal,BaselineStatPolicy>>())
```

Comment:
nit
```suggestion
        auto algo =
        std::make_unique<AlgoType>())
```

Reply:
@noboruma the "auto" make sense, but not the "AlgoType". AlgoType is IMastersSelectionBiasAlgorithm<Decimal, BaselineStatPolicy> which is a abstract class. The point of this line is to select one of two policies as the default.